packages/gg/gg.0.8.0/opam

```diff
@@ -19,4 +19,4 @@ depends: [
   "ocamlfind"
   "ocamlbuild" {build}
 ]
-ocaml-version: [>= "4.00.0"]
+ocaml-version: [>= "4.00.0" & ocaml-version < "4.06.0"]
```

Comment:
that's not passing `opam lint`, or is it?  it should instead be: `available: [ ocaml-version >= "4.00.0" & ocaml-version < "4.06.0" ]`.  same below.  thx!

Reply:
Facepalm.